spec/acceptance/itk_spec.rb

```diff
@@ -1,27 +1,20 @@
 require 'spec_helper_acceptance'
 
-case fact('osfamily')
-when 'Debian'
+case os[:family]
+when 'debian'
   service_name = 'apache2'
-  majrelease = fact('operatingsystemmajrelease')
-  variant = if ['6', '7', '10.04', '12.04'].include?(majrelease)
-              :itk_only
-            else
-              :prefork
-            end
-when 'RedHat'
-  unless fact('operatingsystemmajrelease') == '5'
+  variant = :prefork
+when 'redhat'
+  unless os[:release].to_i == 5
```

Comment:
Why is this here? Shouldn't it be `variant = os[:release].to_i >= 7 ? :prefork : :itk_only`

Reply:
@ekohl yes, you're right. The current implementation is pretty messed up. It can be changed with your solution ;)